src/Psalm/Internal/CallMap.php

```diff
@@ -357,7 +357,7 @@
 'ArithmeticError::getTrace' => ['array<int,array<string,mixed>>'],
 'ArithmeticError::getTraceAsString' => ['string'],
 'array_change_key_case' => ['array|false', 'input'=>'array', 'case='=>'int'],
-'array_chunk' => ['array[]', 'input'=>'array', 'size'=>'int', 'preserve_keys='=>'bool'],
```

Comment:
I missed the $preserve_keys=true case when adding this to Phan. For psalm, two separate signatures may make sense

Reply:
Good catch. I'll revert for now.

Reply:
I forgot what array_chunk was doing, the original version shouldn't have needed changes. The **inner arrays** are lists (`list<list<...>>`) if preserve_keys is false, but they're not lists if preserve_keys is true.

```
php > var_export(array_chunk([1,2,3], 2, true));
array (
  0 => 
  array (
    0 => 1,
    1 => 2,
  ),
  1 => 
  array (
    2 => 3,  // not a list
  ),
)
```